test/api/unit/models/build.test.js

```diff
@@ -52,16 +52,44 @@ describe('Build model', () => {
   });
 
   describe('after create hook', () => {
+    it('should send a build new build message', (done) => {
```

Comment:
Thoughts on using the newer `async/await` syntax here to be consistent with other new tests?

Reply:
it appears implementing this change affects many of the other test because of the affects on the SQS stub ... i'd rather wait to overhaul this test file